microshift_install/microshift-install-rpm.adoc

```diff
@@ -1,8 +1,62 @@
 :_content-type: ASSEMBLY
 [id="microshift-install-rpm"]
-= Installing MicroShift from RPM
+= Installing MicroShift from RPM package
 include::_attributes/attributes-microshift.adoc[]
 :context: microshift-install-rpm
 toc::[]
 
-// The content of this file should be based on https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8.md but without the steps for building from source.
+You can install MicroShift from an RPM package on a machine with {op-system-first} {op-system-version}. 
+
+To embed MicroShift in a {op-system} image, see xref:../microshift_install/microshift-install-rhel-for-edge.adoc#microshift-install-rhel-for-edge[Embedding MicroShift into a {op-system} image]
+
+[id="system-requirements-installing-microshift"]
+== System requirements for installing MicroShift 
+
+The following conditions must be met prior to installing MicroShift: 
+
+* {op-system-first} {op-system-version}
+* 2 CPU cores 
+* 2 GB of RAM 
+* 10 GB of storage 
+* You have an active {product-title} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
+* You have a subscription that includes MicroShift RPMs. 
+* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload. For more information, see xref:../microshift_install/microshift-install-rpm.adoc#preparing-install-microshift-from-rpm-package_microshift-install-rpm[Preparing your {op-system} machine for MicroShift]. 
+
+.Architectures supported on MicroShift
```

Comment:
Is "supported _on_" correct here (as MicroShift is "running on" a CPU / an OS)?

Reply:
@fzdarsky Can we just say 

"MicroShift supported architectures" 

?